src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
Yes, of course. A lot of the problem on the source was caused from using null pointer and calling on them reinterpret_cast to another type. A lot of headaches in V8 sources was for the same exact reason. The only semiworkaround was to use something different from nullptr or NULL. The version of clang is 3.4.1 on my system, that's seems quite old to me. I will try to update it first and then report back.